pkg/controller/operands/kubevirt.go

```diff
@@ -265,16 +286,21 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtv1.KubeVirtConfigurati
 }
 
 func getObsoleteCPUConfig(hcObsoleteCPUConf *hcov1beta1.HyperConvergedObsoleteCPUs) (map[string]bool, string) {
+	obsoleteCPUModels := make(map[string]bool)
+	for _, cpu := range hardcodedObsoleteCPUModels {
+		obsoleteCPUModels[cpu] = true
```

Comment:
why are you setting all keys on `obsoleteCPUModels` map as `true`? is there a scenario in which a cpu listed in `obsoleteCPUModels` will be `false`?

Reply:
KubeVirt API - map of trues